### ISSUE TITLE
PR AUC metric

### ISSUE THREAD

**Issue by annaveronika** [catboost/catboost#737]

**Comment by annaveronika** [comment]
the metric is implemented and will be out in the next minor release

**Comment by tohich51** [comment]
Does this metric has the same evaluation principe as auc(sklearn.precision_recall_curve) ?

**Comment by prasoon2211** [comment]
Since this issue if a top Google search result, to anyone who sees this in the future, you can use:
`eval_metric="PRAUC:use_weights=false"`
as an argument to the `CatBoostClassifier` to get the same behaviour as sklearn (i.e., your P&R will not be weighed when calculating the AUC).

The implementation lives here: https://github.com/catboost/catboost/blob/master/catboost/libs/metrics/metric.cpp

Release notes: https://github.com/catboost/catboost/blob/master/RELEASE.md#new-functionality-5
